lametro/templates/lametro/event.html

```diff
@@ -158,20 +203,25 @@ <h4>Agenda upload</h4>
 {% block extra_js %}
     <script>
     $(document).ready(function() {
-        $('#agendaModal').on('shown.bs.modal', function (e) {  
-            e.preventDefault()
-            agenda_url_submitted = $('#agenda_url').val()
-            agenda_url_clean = full_text_doc_url(agenda_url_submitted)
+        // Admin view: Display preview of event document submitted via URL
+        $('#url_form_modal').on('shown.bs.modal', function (e) {  
+            e.preventDefault();
+            agenda_url_submitted = $('#agenda_url').val();
+            base_url = '{{base_url}}'
+            agenda_url_clean = full_text_doc_url_js(agenda_url_submitted, base_url);
             document.getElementById('pdf-embed-agenda-check').src = agenda_url_clean
         });
 
-        function full_text_doc_url(url) {
-            base_url = '{{base_url}}'
-            encoded_url = encodeURIComponent(url)
-            doc_url = base_url + '?filename=agenda&document_url=' + encoded_url
+        // Admin view: Cancel submission of PDF
+        $('#pdf-form-cancel').on('click', function() {
+            document.getElementById('pdf-check-viewer-test').src = '';
+            document.getElementById('pdf-form-input').value= null;
 
-            return doc_url 
-        };
+            $('#pdf-form-message').addClass('hidden');
```

Comment:
You can select multiple IDs like `$('#pdf-form-message, #pdf-check-viewer-test...').addClass('hidden');`